lib/services/cacheUtils.js

```diff
@@ -91,10 +97,18 @@ function createDomainEnabledCacheHandler(domain, processValueFn, cache, cacheTyp
 function cacheAndHold(cache, cacheType, cacheKey, retrieveRequestFn, processValueFn, callback) {
     var cachedValue = cache.data[cacheType].get(cacheKey);
 
+    function getCacheEventId() {
+        return cacheType + ':' + cacheKey;
+    }
+
     if (cachedValue[cacheKey]) {
```

Comment:
It seems weird but yes, it will. The "get()" function of a nodeCache cache returns an empty object if the key does not exist, so cachedValue will always be an object.